recipes/libtiff/all/conanfile.py

```diff
@@ -24,7 +24,7 @@ class LibtiffConan(ConanFile):
         "fPIC": [True, False],
         "lzma": [True, False],
         "jpeg": [False, "libjpeg", "libjpeg-turbo", "mozjpeg"],
-        "zlib": [True, False],
+        "zlib": [False, "zlib", "zlib-ng"],
```

Comment:
This change will break users that had ``zlib=True`` option defined. I'd try to leave it True with meaning "zlib" and just add the new one.